File: freecodecamp/daily-code-challenge/2026-04-08.py
def is_fizz_buzz(arr):
    start = None

    # Find starting number
    for i, val in enumerate(arr):
        if isinstance(val, int):
            start = val - i
            break

    if start is None:
        return False

    for i in range(len(arr)):
        num = start + i

        if num % 15 == 0:
            expected = "FizzBuzz"
        elif num % 3 == 0:
            expected = "Fizz"
        elif num % 5 == 0:
            expected = "Buzz"
        else:
            expected = num

        if arr[i] != expected:
            return False

    return True

File: freecodecamp/daily-code-challenge/2026-04-08.js
function isFizzBuzz(arr) {
  // Find the starting number
  let start = null;

  for (let i = 0; i < arr.length; i++) {
    if (typeof arr[i] === "number") {
      start = arr[i] - i;
      break;
    }
  }

  if (start === null) return false;

  for (let i = 0; i < arr.length; i++) {
    let num = start + i;

    let expected =
      num % 15 === 0 ? "FizzBuzz" :
      num % 3 === 0 ? "Fizz" :
      num % 5 === 0 ? "Buzz" :
      num;

    if (arr[i] !== expected) {
      return false;
    }
  }

  return true;
}